webapp/src/redux/ducks/studyMaps.ts

```diff
@@ -622,7 +621,23 @@ export default createReducer(initialState, (builder) => {
     })
     .addCase(deleteStudyMapNodeTemp, (draftState, action) => {
       const { studyId, nodeId } = action.payload;
-      delete draftState.entities[studyId]?.nodes[nodeId];
+      const entity = draftState.entities[studyId];
```

Comment:
Why do you change `delete draftState.entities[studyId]?.nodes[nodeId];`?

Reply:
for better readability